frontend/packages/knative-plugin/integration-tests/support/pageObjects/global-po.ts

```diff
@@ -238,6 +238,15 @@ export const eventingPO = {
     searchCatalog: '[data-test="search-catalog"]',
     eventSourceCard: '[data-test="item knative-event-source"]',
   },
+  events: {
+    pageTitle: '[data-test-id="resource-title"]',
```

Comment:
You can define this [here](https://github.com/openshift/console/pull/14417/files#diff-6006aac3bef608898c737798b55d7feb1bcd63eb08952c79c9980d5909b20efaR363) as it's used in the file under dev console package